tnt/src/main/java/org/dcsa/conformance/standards/tnt/checks/TntSchemaConformanceCheck.java

```diff
@@ -0,0 +1,79 @@
+package org.dcsa.conformance.standards.tnt.checks;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.dcsa.conformance.core.check.ActionCheck;
+import org.dcsa.conformance.core.check.JsonSchemaValidator;
+import org.dcsa.conformance.core.traffic.ConformanceExchange;
+import org.dcsa.conformance.core.traffic.HttpMessageType;
+import org.dcsa.conformance.standards.tnt.action.TntEventType;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class TntSchemaConformanceCheck extends ActionCheck {
+
+  private final Map<TntEventType, JsonSchemaValidator> eventSchemaValidators;
+  public TntSchemaConformanceCheck(
+    Predicate<String> isRelevantForRoleName,
+    UUID matchedExchangeUuid,
+    HttpMessageType httpMessageType,
+    Map<TntEventType, JsonSchemaValidator> eventSchemaValidators) {
+    super("", "The HTTP %s matches the standard JSON schema"
+      .formatted(HttpMessageType.RESPONSE.name().toLowerCase()), isRelevantForRoleName,
+      matchedExchangeUuid, httpMessageType);
+    this.eventSchemaValidators = eventSchemaValidators;
+  }
+
+  private boolean isEventNode(JsonNode jsonNode) {
+    return jsonNode.isObject() && !jsonNode.path("eventType").isMissingNode();
+  }
+
+  private ArrayList<JsonNode> _findEventNodes(
```

Comment:
Please use the default naming convention of Java, so no `_` before method names, just camel Casing. 
See https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
done